com/win32com/__init__.py

```diff
@@ -104,8 +104,6 @@ def __PackageSupportBuildPath__(package_path):
 			__gen_path__ = os.path.join(
 								win32api.GetTempPath(), "gen_py",
 								"%d.%d" % (sys.version_info[0], sys.version_info[1]))
```

Comment:
You don't want to touch this file at all, right?

Reply:
Correct, not with this PR since this PR should only contain the GIL state changes. For the gencache topic I'll open another discussion.

Reply:
But this PR does still touch it! :)

Reply:
Sorry I interpreted it wrongly. With the "this file" I thought of the touched file in the implementation, not the __init__.py itself. Hence I reverted the function not the file content itself.